### PR TITLE
Created start screen that will route to either the landingScreen or the dashboard based on the authorization status

### PR DESCRIPTION
**Pull request by dhancockFigure**
The dashboard screen and landing screen should never be in the widget tree at the same time

### CODE REVIEW

File: lib/screens/start_screen.dart [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+class StartScreenState extends State<StartScreen> {
+  late LocalAuthHelper _localAuthHelper;
+
+  @override
+  void initState() {
+    super.initState();
+    _localAuthHelper = get<LocalAuthHelper>();
+    _localAuthHelper.auth(context);
```
Comment (jakegraves-figure) [inline lib/screens/start_screen.dart:21]: So, does this still do the auto-navigation to the dashboard?

**Review by jakegraves-figure — COMMENTED** [review]
Just one little question.

**Review by royd — APPROVED** [review]

File: lib/screens/start_screen.dart (hunk 1 above)
Comment (dhancockFigure) [reply lib/screens/start_screen.dart:21]: no. start_screen will listen to the status stream and route accordingly. Nothing should need to push the landing/dashboard

**Review by jakegraves-figure — APPROVED** [review]

File: lib/screens/start_screen.dart (hunk 1 above)
Comment (dhancockFigure) [reply lib/screens/start_screen.dart:21]: sorry, misread the question. yes, the stream builder will auto navigate based on the status value

**Pull request merged**